### PR TITLE
[Bug 1143951] Send PMs to users from top contributors page.

### PR DESCRIPTION
**Pull request by mythmon**
r?

### CODE REVIEW

File: kitsune/community/static/js/ContributorsList.jsx [hunk 1]
```diff
@@ -21,7 +27,10 @@ export default class ContributorsList extends React.Component {
                 columns={this.props.columns}
                 filters={filters}
                 setFilters={setFilters}
-                allowedOrderings={allowedOrderings}/>
+                allowedOrderings={allowedOrderings}
+                actions={[
+                    {icon: 'paper-plane', onClick: this.sendMessage.bind(this), hover: 'Send PM'},
```
Comment (rlr) [inline kitsune/community/static/js/ContributorsList.jsx]: :airplane:

File: kitsune/community/static/less/community-new.less [hunk 2]
```diff
@@ -6,37 +6,73 @@
...
-        tr {
-            height: 40px;
+        [data-column="select"] {
+            text-align: center;
+        }
+
+        [data-column="actions"] {
+            padding-right: 10px;
+            text-align: center;
+            width: 0; // As small as possible.
```
Comment (rlr) [inline kitsune/community/static/less/community-new.less]: I guess that's a valid comment in less?

Comment (mythmon) [inline kitsune/community/static/less/community-new.less]: Yeah. This is actually an old line, I just re-indented a lot of stuff here. Sorry about that.

**Comment by rlr** [comment]
works nice. I like the UI :fire:

**Pull request merged**